keycloak/realm/realm-export.json

```diff
@@ -1465,8 +1465,8 @@
     "port": "1025",
     "host": "mailhog",
     "replyTo": "",
-    "from": "test@self-serve-portal.com",
-    "fromDisplayName": "",
+    "from": "noreply@unifiedid.com",
+    "fromDisplayName": "UID2 Service (do not reply)",
```

Comment:
is it standard practice to have the "do not reply" in the display name? I feel like the usual pattern would be the display name is just `UID Service` and the "no reply" is in the email address like you already have.

Reply:
I think we are asked to use this display name, can't find the slack message for that anymore, but this is consistent with the email we currently send out from portal (the email send to approver)